modules/kubernetes-addons/ingress-nginx/values.yaml

```diff
@@ -2,10 +2,8 @@ controller:
   service:
     externalTrafficPolicy: "Local"
     annotations:
-      # AWS Load Balancer Controller Annotations
-      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp # or 'ssl'
-      service.beta.kubernetes.io/aws-load-balancer-attributes: load_balancing.cross_zone.enabled=true
+      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
       service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '60'
-      service.beta.kubernetes.io/aws-load-balancer-type: 'external'
-      service.beta.kubernetes.io/aws-load-balancer-scheme: 'internet-facing' # or 'internal'
-      service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: 'ip'
+      service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
+      service.beta.kubernetes.io/aws-load-balancer-internal: "false"
```

Comment:
This annotation is now deprecated. You might need to use this instead `service.beta.kubernetes.io/aws-load-balancer-scheme: internal` 

Valid values are `internal`, `internet-facing`